### PR TITLE
Bump version to 0.5.1

### PR DESCRIPTION
**Pull request by selwin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,4 @@
+<<<<<<< HEAD
```
Comment (mattrobenolt) [inline README.md]: This has an unresolved merge conflict, which is also reflected on pypi. :(

**Comment by mattrobenolt** [comment]
Why does this have 250+ commits? And it causes tests to fail. :( Can this be squashed since it's super hard to figure out what's going on here.

**Comment by mattrobenolt** [comment]
This looks like a bad merge imo since these are all commits that have already existed in the repo.

**Comment by mattrobenolt** [comment]
Seems the only commit we want is https://github.com/selwin/uap-python/commit/b6c3dc9b017f563fb408de9df29af2f782db81a7 ?

**Comment by selwin** [comment]
Do you mind cherry picking the commit into master?

**Comment by mattrobenolt** [comment]
Yeah, I can do that. I'll create a tag too.

Do you mind giving me permission to pypi so I can upload versions and prevent this from happening again?

**Comment by mattrobenolt** [comment]
Pulled in with https://github.com/ua-parser/uap-python/commit/ef275f556950f1ea7cfa38dcb0a4a595008f0a02

**Comment by selwin** [comment]
I don't mind, sorry that I've been too busy to properly maintain this. Is there a way to do it on PyPI?

**Comment by mattrobenolt** [comment]
Oh, no worries at all, this is what open source is about. :heart:

You can go to https://pypi.python.org/pypi?:action=role_form&package_name=ua-parser

Then add me as a Maintainer. My username on pypi is `mattrobenolt`.

Thanks!

**Comment by selwin** [comment]
Done :)


Win~

**Comment by mattrobenolt** [comment]
Awesome, thanks so much. :)

**Pull request closed without merging**